Dockerfile.dev

```diff
@@ -171,6 +171,10 @@ WORKDIR $TVM_DIR/build
 RUN $CMAKE_DIR/bin/cmake $TVM_DIR -DUSE_ILAVTA_CODEGEN=ON -DUSE_ILAFLEX_CODEGEN=ON && \
     make -j"$(nproc)"
 
+# 3la-vta-testbench
+WORKDIR $WORK_ROOT
+RUN git clone git@github.com:uwsampl/3la-vta-testbench.git --recursive
```

Comment:
What TVM commit is required? Or is it that the TVM will have to be built within the directory?

Reply:
It will pull the latest version of `3la-tvm` (or we can specify in the Makefile)